Aptoide/aptoide-client-v8
# [APP-2468] - Update search screen UI

**What does this PR do?**

Updates the search screen UI according to the design on figma. Missing design on results list is due to LargeAppItem and InstallViewShort changes needed.

P.S. FocusedBorderThickness on the search bar is 2.dp instead of 1.dp but this was approved by Silvana.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] Asterisk.kt
- [ ] Games.kt
- [ ] Search.kt
- [ ] SearchView.kt
- [ ] AppIcons.kt

**How should this be manually tested?**

Test the search and see if every asset and item is displayed correctly with the right icons, sizes, fonts, paddings and colors.
  Flow on how to test this or QA Tickets related to this use-case: [APP-2468](https://aptoide.atlassian.net/browse/APP-2468)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2468](https://aptoide.atlassian.net/browse/APP-2468)


**Code Review Checklist**
